Index: coredumper-1.2.1/src/elfcore.c
===================================================================
--- coredumper-1.2.1.orig/src/elfcore.c
+++ coredumper-1.2.1/src/elfcore.c
@@ -40,6 +40,7 @@ extern "C" {
 #include <elf.h>
 #include <fcntl.h>
 #include <limits.h>
+#include <linux/sched.h>
 #include <pthread.h>
 #include <signal.h>
 #include <stdint.h>
Index: coredumper-1.2.1/src/linuxthreads.c
===================================================================
--- coredumper-1.2.1.orig/src/linuxthreads.c
+++ coredumper-1.2.1/src/linuxthreads.c
@@ -40,6 +40,7 @@ extern "C" {
 
 #include <asm/stat.h>
 #include <sched.h>
+#include <linux/sched.h>
 #include <signal.h>
 #include <stdlib.h>
 #include <string.h>
